README.md

```diff
@@ -0,0 +1,41 @@
+Node Tachyon
+============
+
+A local Tachyon server to help with development of the main plugin and app:
+
+[WordPress plugin](https://github.com/humanmade/tachyon-plugin)
+
+[Tachyon app](https://github.com/humanmade/tachyon)
+
+## Installation
+
+Make sure you have libvips installed, the easiest way is to use homebrew:
+
+```
+brew install homebrew/science/vips --with-webp --with-graphicsmagick
```

Comment:
Well done for getting it running under OSX! It's a bit easier until linux, just `npm install`